docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -7,34 +7,61 @@ A {dfanalytics} configuration object has the following properties:
 
 `analysis`::
   (object) The type of analysis that is performed on the `source`. For example: 
-  `outlier_detection`.
+  `outlier_detection`. For more information, see <<dfanalytics-types>>.
+  
+`analyzed_fields`::
+  (object) You can specify both `includes` and/or `excludes` patterns. If 
+  `analyzed_fields` is not set, only the relevant fileds will be included. For 
+  example all the numeric fields for {oldetection}.
 
 `dest`::
-  (object) The destination configuration of the analysis, consisting of `index` 
-  and optionally `results_field` (`ml` by default). 
+  (object) The destination configuration of the analysis. For more information, 
+  see <<dfanalytics-dest-resources>>.
 
 `id`::
   (string) The unique identifier for the {dfanalytics-job}. This identifier can 
   contain lowercase alphanumeric characters (a-z and 0-9), hyphens, and 
   underscores. It must start and end with alphanumeric characters. This property 
   is informational; you cannot change the identifier for existing jobs.
+  
+`model_memory_limit`::
+  (long or string) The approximate maximum amount of memory resources that are 
+  required for analytical processing. Once this limit is approached, data 
+  pruning becomes more aggressive. Upon exceeding this limit, new entities are 
+  not modelled. The default value for {dfanalytics-jobs} is `1gb`.
+  +
+--
+If you specify a number instead of a string, the units are assumed to be MiB.
+Specifying a string is recommended for clarity. If you specify a byte size unit
+of `b` or `kb` and the number does not equate to a discrete number of megabytes,
+it is rounded down to the closest MiB. The minimum valid value is 1 MiB. If you
+specify a value less than 1 MiB, an error occurs. For more information about
+supported byte size units, see <<byte-units,Common options>>.
+
+If your `elasticsearch.yml` file contains an `xpack.ml.max_model_memory_limit`
+setting, an error occurs when you try to create {dfanalytics-jobs} that have
+`model_memory_limit` values greater than that setting. For more information,
+see <<ml-settings>>.
+--
 
 `source`::
   (object) The source configuration, consisting of `index` and optionally a 
-  `query`.
-
+  `query`. For more information, see <<dfanalytics-source-resources>>.
 
+[float]
+[[dfanalytics-types]]
+==== Analysis types
+  
 [float]
 [[oldetection-resources]]
-==== {oldetection-cap} Configuration Objects 
+===== {oldetection-cap} configuration objects 
 
 An {oldetection} configuration object has the following properties:
 
 `n_neighbors` (Optional)::
-  (integer) Defines the value for up to how many neighbors 
-  {oldetection} should check in the index for a certain point to calculate its 
-  {olscore}. When the value is not set, the system will dynamically detect an 
-  appropriate value.
+  (integer) Defines the value for how many neighbors {oldetection} should check 
+  in the index for a certain point to calculate its {olscore}. When the value is 
```

Comment:
I think the following maybe reads better for the first sentence: it is worth saying *nearest* neighbours and also that each method will use the same value if it is set. 

Defines the value for how many nearest neighbors each method of {oldetection} will use to calculate its {olscore}.